plugins/woocommerce/includes/wc-order-functions.php

```diff
@@ -872,22 +872,39 @@ function wc_order_search( $term ) {
 function wc_update_total_sales_counts( $order_id ) {
 	$order = wc_get_order( $order_id );
 
-	if ( ! $order || $order->get_data_store()->get_recorded_sales( $order ) ) {
+	if ( ! $order ) {
+		return;
+	}
+
+	$recorded_sales  = $order->get_data_store()->get_recorded_sales( $order );
+	$reflected_order = in_array( $order->get_status(), array( 'cancelled', 'trash' ) );
```

Comment:
```suggestion
	$reflected_order = in_array( $order->get_status(), array( 'cancelled', 'trash' ), true );
```

We need to use strict comparisons (the `WordPress.PHP.StrictInArray.MissingTrueStrict` check is currently failing).